WcaOnRails/app/models/incident.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class Incident < ApplicationRecord
+  # Note: *always* insert a new status at the end of the array
+  # 0 (:pending) is the default (see db schema).
+  enum status: [:pending, :solved, :solved_awaiting_digest, :solved_digest_sent]
```

Comment:
Given @Laura-O's other comments I'll just rewrite this to have the following attributes:
 - `solved_at` (datetime), if nil then the incident is pending, if not it has been solved
 - `digest_worthy` (boolean), would be useful to WRC only basically
 - `digest_sent_at` (datetime), to know when it has been set (irrelevant if `digest_worthy` is false). It will also be very useful to automatically prepare digests.